src/main/webapp/script.js

```diff
@@ -0,0 +1,33 @@
+function listElections() {
+  let stateSelected = document.getElementById('select-state').value;
+  let nationalElections = [];
+  let stateElections = [];
+
+  if (stateSelected === null) {
+    return;
+  }
+
+  fetch('/election')
+    .then(response => response.json())
+    .then((textResponse) => {
+      let electionList = JSON.parse(textResponse).elections;
+      electionList.forEach((election) => {
+        if (election.ocdDivisionId == stateSelected) {
+          // TODO: update ocdDivisionId to cater to state vs. national elections
+          stateElections.append(election);
+        }
+      });
+      
+      buildElectionListElements(stateElections);
+  });
+}
+
+function buildElectionListElements(elections) {
```

Comment:
Does the 'elections' parameter ever get used in this function?

Reply:
Again leave a TODO if it's going to get used later but you just have this here right now.

Reply:
I just switched this implementation to Handlebars, so I won't be using this function anymore!